martide/airports
# feat(WEB-7334): Add API for getting timezone based on IATA code

To test:
1. Run `mix airports.update`
2. It should update the timezone_id column of airports_with_tz.csv
3. Run `iex -S mix`
4. Command `Airports.get_timezone("LHR")`. Or change LHR to any airport. It should return the timezone
